### ISSUE TITLE
Unit stats triggers 

### ISSUE THREAD

**Issue by 1ps3** [WeakAuras/WeakAuras2#386]
Would be nice to see some triggers for character stats.

**Comment by InfusOnWoW** [comment]
Can you add a few examples on what users are checking for?

I certainly have seen some users wanting something like that, but a small list of examples would help ensure that it solves the users problems.

**Comment by 1ps3** [comment]
Mostly stuff like movement speed, critical strike, etc. with operators for the trigger option. I know most stats so I guess the selection so the trigger should probably be a % to make it easier.

**Comment by Stanzilla** [comment]
meh. How many usecases are there for it? I understand wanting to display those stats but as a trigger?

**Comment by 1ps3** [comment]
Even if the trigger was used to display stats the only thing you'd want to see them is to see breakpoints during combat. There's a few classes that need it every tier, like fire mage used to, balance Druid before the helm nerf are off the top of my head. So I do kind of think adding this would just make it easier for people to display this kind of info without needing custom.

**Comment by Hamsda** [comment]
I agree, it might be worth adding. There are always some use cases (people needing to see breakpoints or maybe just displaying the stats for streams /shrug) and making it easier for people not to use custom code is a good enough reason in my book. 
That's part of the reason why stuff like the range trigger got added as well, isn't it?

I could imagine it in two different ways:
a) a new dropdown in the status triggers, where you have a page with fields for each stat with percent and rating each and you just check each box that is needed and enter the percent/rating
b) a new field in the unit characteristics page, with a dropdown for which stat, then one for percent/rating, and then a textbox to enter the rating stat.

Not sure which would be better, but both have their advantages.

**Comment by Stanzilla** [comment]
But the custom auras for stuff like this are super easy, custom event and find the one for your stat and almost done.

**Comment by Hamsda** [comment]
True, but so are the custom replacements for many of the default triggers, one to three lines of lua code and done. It's just much nicer for many users we see every day in the discord to not have to use anything custom at all.

Still, up to you guys :)

**Comment by InfusOnWoW** [comment]
Also consider adding armor checks, for inspiration see https://github.com/WeakAuras/WeakAuras2/issues/1065

**Comment by emptyrivers** [comment]
This has now been implemented, after sitting in limbo for a year.
